package.json

```diff
@@ -46,6 +46,7 @@
     "react-dom": "^17.0.1",
     "react-i18next": "^11.7.3",
     "react-query": "^3.2.0",
+    "react-router-dom": "^5.2.0",
```

Comment:
Can't we just add `react-router` as a dependency instead of `react-router-dom`?

Reply:
Didn't know that was possible, I just followed https://reactrouter.com/web/guides/quick-start
Thanks for the tip